src/gui/element.rs

```diff
@@ -1386,6 +1394,12 @@ pub fn menu_container<'a>(
     classic_ptr_btn_state: &'a mut button::State,
     self_update_state: &'a mut SelfUpdateState,
 ) -> Container<'a, Message> {
+    let flavor = config.wow.flavor;
+
+    // State.
+    let default = &State::default();
+    let myaddons_state = state.get(&Mode::MyAddons(flavor)).unwrap_or(default);
```

Comment:
I think this feels more idiomatic, since we can used cloned to get a Option<State> then use unwrap_or_default. It would also be worthwhile adding derive(Copy) on State since its sized. We should always try to derive Copy along w/ Clone where we can

```diff
-    let default = &State::default();
-    let state = state.get(&Mode::MyAddons(flavor)).unwrap_or(default);
+    let state = state
+        .get(&Mode::MyAddons(flavor))
+        .cloned()
+        .unwrap_or_default();
```